lib/opentracing/span.rb

```diff
@@ -0,0 +1,59 @@
+module OpenTracing
+  # Span represents an OpenTracer Span
+  #
+  # See http://www.opentracing.io for more information.
+  class Span
+    NOOP_INSTANCE = Span.new.freeze
+    # Name of the operation
+    attr_accessor :operation_name
```

Comment:
Looking at the Go library it looks like there is not supposed to be a getter for this attribute, so removing this `attr_accessor` and adding a `#operation_name=(name)` method seems like the right way to go.